crates/virtio-queue/src/chain.rs

```diff
@@ -389,21 +388,6 @@ mod tests {
     fn test_indirect_descriptor_err() {
         // We are testing here different misconfigurations of the indirect table. For these error
         // case scenarios, the iterator over the descriptor chain won't return a new descriptor.
-        {
```

Comment:
I think we should keep the test and assert that it is okay to have an address that is not aligned. We should also move the test somewhere else since this test function is for indirect descriptor errors.

Reply:
Thanks for reviewing, I add a new case for indirect descriptor with unaligned address.